boards/arm/b_l4s5i_iot01a/b_l4s5i_iot01a.dts

```diff
@@ -49,3 +49,104 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+&i2c1 {
```

Comment:
Please update doc & yaml to reflect this addition

Reply:
i2c1 is for Arduino